### DISCUSSION TITLE
Require the math used by floris

### DISCUSSION THREAD

**Discussion by HariAB136** [NatLabRockies/floris#730] [Q&A]
Can anyone pls share the link of the pdf which discuss the math involved behind the floris. 
I want to know the formula to calculate power of a wind turbine which is yawed at a particular angle, and many other things along with this

**Comment by paulf81** [comment]
There isn't one paper defining all of FLORIS, but if you look into the files defining the individual wake models you can often find the relavent papers listed there.  These individual references will include citations, which are defined in docs/references.bib

**Reply by paulf81** [reply]
Power loss to a particular yaw angle is discussed in a few papers and is the subject of current research.   See for example:

Simley, E., Fleming, P., Girard, N., Alloin, L., Godefroy, E., and Duc, T.: Results from a wake-steering experiment at a commercial wind plant: investigating the wind speed dependence of wake-steering performance, Wind Energ. Sci., 6, 1427–1453, https://doi.org/10.5194/wes-6-1427-2021, 2021.

And for current research see:

https://www.cambridge.org/core/journals/journal-of-fluid-mechanics/article/modelling-the-induction-thrust-and-power-of-a-yawmisaligned-actuator-disk/3A34FC48A6BC52A78B6D221C13F4FC3A

for one example
